mylyn.context/org.eclipse.mylyn.cdt.tests/data/TestProject/.project

```diff
@@ -5,23 +5,7 @@
 	<projects>
 	</projects>
 	<buildSpec>
-		<buildCommand>
```

Comment:
What was cdt doing in Mylyn?

Reply:
We have the org.eclipse.mylyn.cdt-feature feature!